docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/POST/ProductView.json.example

```diff
@@ -7,96 +7,53 @@
-                                {
-                                    "_media-type": "application/vnd.ibexa.api.AttributeAssignment+json",
-                                    "is_required": false,
-                                    "is_discriminator": false,
-                                    "Attribute": {
-                                        "_media-type": "application/vnd.ibexa.api.Attribute+json",
-                                        "identifier": "measure",
-                                        "name": "measure",
-                                        "description": "",
-                                        "type": "Measurement",
-                                        "type_identifier": "measurement",
-                                        "group": "Test Attribute group",
-                                        "position": 0,
-                                        "options": {
-                                            "type": "length",
-                                            "unit": "centimeter",
-                                            "measurementRange": {
-                                                "minimum": 12,
-                                                "maximum": 15
-                                            },
-                                            "measurementDefaultRange": {
-                                                "defaultRangeMinimumValue": null,
-                                                "defaultRangeMaximumValue": null
-                                            }
-                                        }
-                                    }
-                                },
-                                {
-                                    "_media-type": "application/vnd.ibexa.api.AttributeAssignment+json",
-                                    "is_required": false,
-                                    "is_discriminator": false,
-                                    "Attribute": {
-                                        "_media-type": "application/vnd.ibexa.api.Attribute+json",
-                                        "identifier": "leather_color",
-                                        "name": "leather color",
-                                        "description": "",
-                                        "type": "Color",
-                                        "type_identifier": "color",
-                                        "value": "#000000",
-                                        "group": "Test Attribute group",
-                                        "position": 0,
-                                        "options": []
-                                    }
-                                }
-                            ]
+                            "identifier": "desk",
+                            "name": "Desk",
+                            "AttributeAssignmentList": []
```

Comment:
Attribute assignment lists are suddenly empty in examples. I assume it was not intentonal?

Reply:
Yes, bad copying, will fix